src/FomaClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Cimpress.Clients.Foma.Model;
+using Cimpress.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Cimpress.Clients.Foma
+{
+    public class FomaClient
+    {
+        private readonly HttpClient httpClient;
+        private readonly ILogger logger;
+        private const string baseUrl = "https://fulfillment.at.cimpress.io";
+
+        public FomaClient(HttpClient httpClient, ILogger logger)
+        {
+            this.httpClient = httpClient;
+            this.logger = logger;
+        }
+
+        public async Task<NotificationsResultDto> GetNotifications(string fulfillerId)
+        {
+            var url = $"{baseUrl}/v1/notifications?fulfillerId={fulfillerId}";
+            return await GetData<NotificationsResultDto>(url);
+        }
+
+        public async Task<T> GetData<T>(string url)
+        {
+            using (var response = await httpClient.GetAsync(url))
+            {
+                await response.LogAndThrowIfNotSuccessStatusCode(logger);
+                return await response.Content.ReadAsAsync<T>();
+            }
+        }
+
+        public async Task SendData<T>(string url, T data = default(T))

```

Comment:
The method name does not read very naturally, especially when it's used to accept/reject a notification. I do not have a better proposal though, maybe we can introduce more specific methods later?